README.md

```diff
@@ -225,48 +165,22 @@ Example:
 
```

Comment:
Great job @jorgegoco 🤝

- [ ] [Optional] You may remove the second author from your readme file since it is only one person who contributed to this pull request.

Reply:
Ok, thanks.